src/wolfgame.py

```diff
@@ -6875,15 +6857,50 @@ def fgame(cli, nick, chan, rest):
             gamemode = gamemode.split()[0]
             gamemode = complete_one_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
             if not gamemode:
-                cli.notice(nick, messages["invalid_mode"].format(rest.split()[0]))
+                wrapper.pm(messages["invalid_mode"].format(message.split()[0]))
                 return
             parts[0] = gamemode
 
-        if cgamemode(cli, "=".join(parts)):
-            cli.msg(chan, messages["fgame_success"].format(nick))
+        if cgamemode("=".join(parts)):
+            channels.Main.send(messages["fgame_success"].format(wrapper.source))
             var.FGAMED = True
     else:
-        cli.notice(nick, fgame.__doc__())
+        wrapper.pm(fgame.__doc__())
+
+@command("frole", flag="d", phases=("join",))
+def frole(var, wrapper, message):
+    """Force a player into a certain role."""
+    pl = get_players()
+    if wrapper.source not in pl and not wrapper.source.is_admin():
+        return
+
+    to_force = {}
+
```

Comment:
I don't like how you made this work. Requiring underscores in role names is not something we do anywhere else (e.g. !fgame roles). Make some sort of punctuation like comma required, and split on that instead of replacing all of that with spaces. Then, we can correctly handle spaces in role names.

Reply:
Yeah this was annoying me too. I'd rather type commas than underscores